src/WebJobs.Extensions.DurableTask/OrchestrationTriggerAttribute.cs

```diff
@@ -12,7 +12,11 @@ namespace Microsoft.Azure.WebJobs
     /// </summary>
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("{Orchestration} ({Version})")]
+#if NETSTANDARD2_0
+    [Binding(TriggerHandlesReturnValue = false)]
```

Comment:
@kashimiz I recall we added this for `ActivityTriggerAttribute` but was there a reason we didn't originally add this to `OrchestrationTriggerAttribute`?

Reply:
This attribute signals to the function indexer that triggers to which it's applied are meant to handle a return value from non-C# functions. Since we were focused on a regular functions scenario, missing this was probably oversight.

Reply:
Interestingly enough, the associated Issue (#120) happens when I'm using the Client, not the Trigger...

Reply:
We do in the future want orchestration triggers to handle return values in non-C# scenarios, so this should be true instead of false.